sdk/react/package.json

```diff
@@ -7,7 +7,7 @@
         "clean": "rimraf dist",
         "test": "jest",
         "tsc": "tsc",
-        "build": "yarn clean && yarn tsc"
+        "build": "pnpm clean && pnpm tsc"
```

Comment:
Why is there even a build command here? shouldn't it be using `nx`?

Reply:
uh ya